README.md

```diff
@@ -1,2 +1,12 @@
 # EAL_Shadow_Figures
-Migration of an EAL website to DPUL
+
+A Python module that generates a web site to expose photographs of
+items in an EAL collection of Chinese shadow figures.
+
+## Installation
+  * Clone the git repository
+  * import the images
+  * run poetry install
+
+## Usage
+   poetry run python build-site.py
```

Comment:
I don't see an `index.html` file — it looks like any of the html files in `site` work fine, but should there be a home page?

Reply:
See the PR notes: I'm assuming we're going to want to wrap a Jekyll site around it?  If not, I can create the site scaffolding too.